storage/pkg/flagset/storagehomedata.go

```diff
@@ -1,118 +0,0 @@
-package flagset
-
-import (
-	"github.com/micro/cli/v2"
-	"github.com/owncloud/ocis/storage/pkg/config"
-)
-
-// StorageHomeDataWithConfig applies cfg to the root flagset
-func StorageHomeDataWithConfig(cfg *config.Config) []cli.Flag {
-	flags := []cli.Flag{
-
-		// debug ports are the odd ports
-		&cli.StringFlag{
-			Name:        "debug-addr",
-			Value:       "0.0.0.0:9157",
-			Usage:       "Address to bind debug server",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_DATA_DEBUG_ADDR"},
-			Destination: &cfg.Reva.StorageHomeData.DebugAddr,
-		},
-
-		// Services
-
-		// Storage home data
-
-		&cli.StringFlag{
-			Name:        "network",
-			Value:       "tcp",
-			Usage:       "Network to use for the storage service, can be 'tcp', 'udp' or 'unix'",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_DATA_NETWORK"},
-			Destination: &cfg.Reva.StorageHomeData.Network,
-		},
-		&cli.StringFlag{
-			Name:        "protocol",
-			Value:       "http",
-			Usage:       "protocol for storage service, can be 'http' or 'grpc'",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_DATA_PROTOCOL"},
-			Destination: &cfg.Reva.StorageHomeData.Protocol,
-		},
-		&cli.StringFlag{
-			Name:        "addr",
-			Value:       "0.0.0.0:9156",
-			Usage:       "Address to bind storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_DATA_ADDR"},
-			Destination: &cfg.Reva.StorageHomeData.Addr,
-		},
-		&cli.StringFlag{
-			Name:        "url",
-			Value:       "localhost:9156",
-			Usage:       "URL to use for the storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_DATA_URL"},
-			Destination: &cfg.Reva.StorageHomeData.URL,
-		},
-		&cli.StringSliceFlag{
-			Name:    "service",
-			Value:   cli.NewStringSlice("dataprovider"),
-			Usage:   "--service dataprovider [--service otherservice]",
-			EnvVars: []string{"STORAGE_STORAGE_HOME_DATA_SERVICES"},
-		},
-		&cli.StringFlag{
-			Name:        "driver",
-			Value:       "owncloud",
-			Usage:       "storage driver for home data mount: eg. local, eos, owncloud, ocis or s3",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_DATA_DRIVER"},
```

Comment:
used in drone https://github.com/owncloud/ocis/blob/d422a91a01b98893d0a10a7555092342755ee33b/.drone.star#L547
not needed anymore?

Reply:
This option does not exist anymore.